ansible/roles/builder/tasks/main.yml

```diff
@@ -28,6 +39,16 @@
   args:
     creates: C:\Python27\Lib\site-packages\lxml\__init__.py
 
+- name: Install Zull (for zuul-cloner)
+  win_command: pip install zuul==2.6.0
```

Comment:
Perhaps we can use the [pip module](http://docs.ansible.com/ansible/latest/pip_module.html) here? (Not sure if it works on windows though)